plutus-core/plutus-ir/test/PlutusIR/Transform/DeadCode/Tests.hs

```diff
@@ -31,3 +35,14 @@ test_deadCode = runTestNestedIn ["plutus-ir/test/PlutusIR/Transform"] $
             , "recBindingComplex"
             , "pruneDatatype"
             ]
+
+test_typecheck :: TestTree
+test_typecheck = testGroup "typechecking"
+  [ testProperty "Builtin Variant1" $
+  withMaxSuccess 3000
```

Comment:
why are we setting this everywhere? is the default setting too low? too high?

Reply:
I was using it to change the max indiviually when sometimes some of them pass locally. With the weaker typechecking it's not needed maybe.